src/main/java/org/broadinstitute/hellbender/utils/help/GATKHelpDoclet.java

```diff
@@ -0,0 +1,111 @@
+    private final static String WALKER_TYPE_MAP_ENTRY = "walkertype";      // populated from javadoc custom tag
+
+    /**
+     * Create a doclet of the appropriate type and generate the FreeMarker templates properties.
+     * @param rootDoc
+     * @throws IOException
+     */
+    public static boolean start(final RootDoc rootDoc) throws IOException {
+        return new GATKHelpDoclet().startProcessDocs(rootDoc);
+    }
+
+    /**
+     * Return the name of the freemarker template to be used for the index generated by Barclay.
+     * Must reside in the folder passed to the Barclay Javadc Doclet via the "-settings-dir" parameter.
+     * @return name of freemarker index template
+     */
+    @Override
+    protected String getIndexTemplateName() {
+        return GATK_FREEMARKER_INDEX_TEMPLATE_NAME;
+    }
+
+    /**
+     * @return Create and return a DocumentedFeatureHandler-derived object to handle documentation
+     * for the target feature(s) for this doclet.
+     */
+    @Override
+    protected DocumentedFeatureHandler createDocumentedFeatureHandler(
+            final ClassDoc classDoc,
+            final DocumentedFeatureObject documentedFeature) {
+        return new GATKHelpDocumentationHandler();
+    }
+
+    /**
+     * Create a GSONWorkUnit-derived object that holds our custom data. This method should create the object, and
+     * propagate any custom javadoc tags from the template map to the newly created GSON object; specifically
+     * "walkertype", which is pulled from a custom javadoc tag.
+     *
+     * @param workUnit
+     * @param groupMaps
+     * @param featureMaps
+     * @return
+     */
+    @Override
+    protected GSONWorkUnit createGSONWorkUnit(
+            final DocWorkUnit workUnit,
+            final List<Map<String, String>> groupMaps,
+            final List<Map<String, String>> featureMaps)
+    {
+        GATKGSONWorkUnit gatkGSONWorkUnit = new GATKGSONWorkUnit();
+        gatkGSONWorkUnit.setWalkerType((String)workUnit.rootMap.get(WALKER_TYPE_MAP_ENTRY));
+        return gatkGSONWorkUnit;
+    }
+
+    //TODO: do we need to retain supercategory, and if so, we may need to update the rules...
+    /**
+     * Trivial helper routine that returns the map of name and summary given the documentedFeatureObject
+     * AND adds a super-category so that we can custom-order the categories in the index
+     *
+     * @param documentedFeatureObject
+     * @return
+     */
+    @Override
+    protected final Map<String, String> getGroupMap(final DocumentedFeatureObject documentedFeatureObject) {
+        final Map<String, String> root = super.getGroupMap(documentedFeatureObject);
+
+        /**
+         * Add-on super-category definitions. The assignments depend on parsing the names
+         * defined in HelpConstants.java so be careful of changing anything.  The super-category
+         * value strings need to be the same as used in the Freemarker template.
+         */
```

Comment:
It is indeed -- it's also a duct tape patch I added years ago for organizing content on the website. Would be delighted to have something better, so suggestions >> welcome.